core/src/commonMain/kotlin/com/ustadmobile/core/controller/ContentEntryList2Presenter.kt

```diff
@@ -178,10 +179,37 @@ class ContentEntryList2Presenter(context: Any, arguments: Map<String, String>, v
         }
     }
 
+    /**
+     * Show ContentEntryList in picker mode so the user can select a folder to move entries to.
+     *
+     * @param selectedContentEntryParentChildJoinUids a string which is a comma separated list of
+     * the selected ContentEntryParentChildJoinUids (e.g. that should be saved to the savedStateHandle)
+     */
+    fun handleShowFolderOnlyEntries(selectedContentEntryParentChildJoinUids: String){
```

Comment:
handleClickMove(childrenToMove: List<ContentEntryWithEtc>)

There is no longer any need to save KEY_SELECTED_ITEMS because the next presenter should not be looking back...

The move should happen when the selection is made (e.g. in ContentEntrySelectFolder or ContentEntryEdit if creating a new folder) - ensure consistent approach.